src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -204,3 +204,437 @@
     text: az cosmosdb keys list --name MyCosmosDBDatabaseAccount --resource-group MyResourceGroup --subscription MySubscription
     crafted: true
 """
+
+helps['cosmosdb sql'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL resources.
+"""
+
+# SQL database
+helps['cosmosdb sql database'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL databases.
+"""
+
+helps['cosmosdb sql database create'] = """
+type: command
+short-summary: Create an SQL database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb sql database list'] = """
+type: command
+short-summary: List the SQL databases under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb sql database show'] = """
+type: command
+short-summary: Get the SQL database under an Azure Cosmos DB database account.
```

Comment:
I think it should be "Show the details/properties of an SQL database under an Azure Cosmos DB database account".